base/base_tests/string_utils_test.cpp

```diff
@@ -1270,3 +1271,18 @@ UNIT_TEST(Trim)
   strings::Trim(str, "tsgn");
   TEST_EQUAL(str, "ri", ());
 }
+
+UNIT_TEST(UniChar_Ostream_Istream)
+{
+  strings::UniChar c1, c2, c3 = 0xabcdef;
+  std::istringstream istreamUpper("ABCDEF");
+  istreamUpper >> c1;
+  std::istringstream istreamLower("00abcdef");
+  istreamLower >> c2;
```

Comment:
@vng we can read and convert unicode properly here, instead of reading from/to hex. WDYT?